src/Microsoft.Azure.WebJobs.Host/Executors/FunctionExecutor.cs

```diff
@@ -618,6 +625,48 @@ private static ITaskSeriesTimer StartParameterLogTimer(IRecurrentCommand updateC
             //      a. If throwOnTimeout, we throw the FunctionTimeoutException.
             //      b. If !throwOnTimeout, wait for the task to complete.
 
+            // Where the invocation filter attribute functionality is currently temporarily placed
+            var functionMethod = instance.FunctionDescriptor.Method;
+            var invokeFilter = functionMethod.GetCustomAttribute<InvocationFilterAttribute>();
+            
+            var parameters = new Dictionary<string, object>();
+            string[] pnames = parameterNames.ToArray<string>();
+
+            for (var i = 0; i < invokeParameters.Length; i++)
+            {
+                parameters.Add(pnames[i], invokeParameters[i]);
+            }
+
+            if (invokeFilter != null)
+            {
+                var functionExecutingContext = new FunctionExecutingContext(instance.Id, instance.FunctionDescriptor.FullName, parameters, logger, host);
+                var cancellationToken = functionCancellationTokenSource.Token;
+                try
+                {
+                    await invokeFilter.OnExecutingAsync(functionExecutingContext, cancellationToken);
+                }
+                catch (Exception e)
+                {
```

Comment:
The intent was to pass any exceptions thrown from filters or the function invocation on to any Executed filters. If we keep that design, you can't just rethrow here, instead you'd have to include all of this in the try/catch you have below.